gate-deck-plugins/src/main/kotlin/com/netflix/spinnaker/gate/plugins/DeckPluginsController.kt

```diff
@@ -44,9 +47,19 @@ class DeckPluginsController(
     @PathVariable pluginId: String,
     @PathVariable pluginVersion: String,
     @PathVariable asset: String
-  ): String {
-    return deckPluginService.getPluginAsset(pluginId, pluginVersion, asset)
-      ?: throw NotFoundException("Unable to find asset for plugin version")
+  ): ResponseEntity<String> {
+    val pluginAsset = deckPluginService.getPluginAsset(pluginId, pluginVersion, asset) ?: throw NotFoundException("Unable to find asset for plugin version")
+
+    return ResponseEntity.ok()
+        .header("Content-Type", pluginAsset.contentType)
+        .header("Cache-Control",
+            CacheControl
+                .maxAge(1, TimeUnit.DAYS)
+                .mustRevalidate()
+                .cachePrivate()
+                .headerValue
```

Comment:
These could be configurable eventually, I figured we can just roll with this for now.

Reply:
We're using `cache-control: max-age=2592000` for regular deck assets at nflx